.velocitas.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packages": {
+    "devenv-runtimes": "v4.0.5",
+    "devenv-github-workflows": "v6.0.4",
+    "devenv-github-templates": "v1.0.5",
+    "devenv-devcontainer-setup": "v2.4.7"
+  },
+  "components": [
+    "devcontainer-setup",
+    "runtime-local",
+    "build-system",
+    "grpc-interface-support"
+  ],
```

Comment:
I just used the basic stuff here, but it make sense to double check this. I guess not all components are useful yet. 

grpc-interface-support is not working yet, as there is no support for language "kotlin"

Reply:
I guess the rest of the packages is also lacking support for this repo, don't know if we need anything at all...